checks/network.py

```diff
@@ -75,6 +75,20 @@ def checkDynamicBitrate(lines):
     return None
 
 
+def checkNetworkOptimizations(lines):
+    networkOptimizationLines = search('New socket loop enabled by user', lines)
+    if (len(networkOptimizationLines) > 0):
+        return [LEVEL_INFO, "Network Optimizations", "Network Optimizations are enabled. This is disabled by default. It is only suggested to enable Network Optimizations if you are having issues streaming. The setting can be toggled in Settings -> Advanced -> Network."]
```

Comment:
`This is` -> `This option is` 

Unrelated to this PR, but to log at least somewhere public, we should have some kind of KB article for these settings that we can add here as links for the "what issues does this help with" part of things.

Reply:
Addressed.